QA/README.md

```diff
@@ -0,0 +1,79 @@
+# CherryTwist - Quality Assurance
+
+This document provides overview of "Cherrytwist" quality assurance and testing activities. Some of them are in progress and some are to be started.
+
+# Process
+ 
+- Perform functional/non-functional testing types on reviewed "github" issues
+    - Depending on the "github" issue scope, testing is performed on different environments 
+- Log defects, communicate  and retest them
+- Perform regression testing as part of "github" issues testing
+- Automate/maintain test cases on different levels
+- Test data management
+
+
+# Testing types and levels
+
+"Cherrytwist" consists of multiple services. The following test types are being applied among the different test levels:
+
+## "Server" - testing types and levels:
+- ### Functional testing: 
+    - Unit testing<sup>*</sup>
+    - [GraphQL API integration tests](https://github.com/cherrytwist/Server/tree/develop/test)
+    - GraphQL Exploratory testing
+        
+- ### Non-functional testing:
+    - [Performance testing](https://github.com/cherrytwist/Server/tree/develop/test/performance)
+    - Security testing<sup>*</sup>
+    - Database migration<sup>*</sup>
+
+- ### Static testing:
+    - Code review
+    - Static code analyses tools<sup>*</sup>
+ 
+## "Client.Web" - testing types and levels:
+- ### Functional testing: 
+    - Component testing<sup>*</sup>
+    - Integration testing<sup>*</sup>
+    - E2E automation testing<sup>*</sup>
+    - Exploratory testing
+        
+- ### Non-functional testing:
+    - Performance testing<sup>*</sup>
+    - Security testing<sup>*</sup>
+
+
+# Reporting
+ 
+- [Test cases](https://travis-ci.com/github/cherrytwist/Server/builds)
+
+    - Number of test cases
+    - Number of failed test cases per service
+
+- Defects<sup>*</sup>
+
+    - Number of defects<sup>*</sup>
+    - Number of production environment defects by severity<sup>*</sup>
+    - Number of development environment defects by severity<sup>*</sup>
+
+# Environments
+- Local - setup for [Server](https://github.com/cherrytwist/Server) and [Client.Web](https://github.com/cherrytwist/Client.Web)
+- [Development](https://dev.cherrytwist.org/) - auto-deployed after each merge to development branch
+- [Test](https://test.cherrytwist.org/) - QA deploys from "github" repositories action, the required branch
+- [Acc](https://acc.odyssey.ninja:3000/ecoverse) - Project manager deploys from "github" repositories action, from master branch 
```

Comment:
Should not refer to Odyssey environments there - they are not part of Cherrytwist but our first client